packages/e2e/README.md

```diff
@@ -27,12 +27,19 @@ void main() {
 }
 ```
 
+## Test locations
+
+It is recommended to put e2e tests in the `test/` folder of the app or package.
+For example apps, if the e2e test references example app code, it should go in
+`example/test/`. It is also acceptable to put e2e tests in `test_driver/` folder
+so that they're alongside the runner app (see below).
+
 ## Using Flutter driver to run tests
 
 `E2EWidgetsTestBinding` supports launching the on-device tests with `flutter drive`.
 Note that the tests don't use the `FlutterDriver` API, they use `testWidgets` instead.
 
-Put the a file named `<package_name>_test.dart` in the app' `test_driver` directory:
+Put the a file named `<package_name>_e2e_test.dart` in the app' `test_driver` directory:
```

Comment:
Does `flutter test` scans `_test.dart` in a directory and execute them?

Reply:
Yes. It only looks in the test/ directory by default.

Reply:
Then when executing `flutter test` without an attached device, the `_e2e_test.dart` will execute. Is it expected?

Reply:
`flutter test` only looks for tests in the test/ folder. test_driver/ is ignored.